src/modules/ConnectionModule.js

```diff
@@ -0,0 +1,70 @@
+const { Module, Connection } = require('../')
+
+module.exports = class ConnectionModule extends Module {
+  constructor (...args) {
+    super(...args)
+    this.name = 'connection'
+  }
+
+  get _users () {
+    return this.client.database.users
+  }
+
+  async connect (_user, _connection, _tokens) {
+    if (typeof _tokens === 'string') _tokens = { token: _tokens }
+    if (typeof _tokens !== 'object') throw new Error(`ConnectionModule.connect should have the last argument a string or an object, ${typeof _tokens} given`)
+
+    console.log('a')
+    //const user = await this._users.get(_user, 'connections')
+    const user = await this._users.findOne(_user)
```

Comment:
```suggestion
    const user = await this._users.findOne(_user, 'connections')
```
And make sure the `UserRepository` has a default value for the `connections` field. (not the Schema, the Repository)

Reply:
done